manticore/native/memory.py

```diff
@@ -577,18 +576,18 @@ class Memory(object, metaclass=ABCMeta):
     This class handles all virtual memory mappings and symbolic chunks.
     """
 
-    def __init__(self, maps=None, cpu=StubCPU()):
+    def __init__(self, maps: Optional[Iterable[Map]] = None, cpu=StubCPU()):
```

Comment:
I wonder... I know you didn't touch this, but shouldn't the `cpu=StubCPU()` be `cpu=None` and then do
```
if not cpu:
    cpu = StubCPU()
```

Reply:
Six of one, half-dozen of another?